sherpa/astro/plot.py

```diff
@@ -112,8 +124,22 @@ def prepare(self, data, stat=None):
         # Maybe to_plot should return the lo/hi edges as a pair
         # here.
         #
+        if isinstance(stat, WStat) and data.background_ids != [] and not data.subtracted:
+            # When using the WStat statistic we want to show the
+            # subtracted data. How are the error-bars handled in
+            # this case?
```

Comment:
That's a questions for @anetasie! However, personally, I find it looks convincing in the plots.

Reply:
We add in fake error bars - using Chi2XspecVar - but I don't know if it just uses the "source" counts or does something fancy to handle combined data.

A quick check suggests that subtracting the background does not change the Y errors (in this case not using WSTAT but I would expect this to be hand;ed generically), which I guess answers this question (we can see that y2 <= y1 but dy2 == dy1):

```
sherpa In [1]: load_pha('3c273.pi')
WARNING: systematic errors were not found in file '3c273.pi'
statistical errors were found in file '3c273.pi'
but not used; to use them, re-read with use_errors=True
read ARF file 3c273.arf
read RMF file 3c273.rmf
WARNING: systematic errors were not found in file '3c273_bg.pi'
statistical errors were found in file '3c273_bg.pi'
but not used; to use them, re-read with use_errors=True
read background file 3c273_bg.pi

sherpa In [2]: d = get_data()

sherpa In [3]: y1 = d.to_plot()[1]

sherpa In [4]: dy1 = d.to_plot()[3]

sherpa In [5]: subtract()

sherpa In [6]: y2 = d.to_plot()[1]

sherpa In [7]: dy2 = d.to_plot()[3]

sherpa In [8]: y2 < y1
Out[8]:
array([ True,  True, False,  True,  True, False, False, False,  True,
        True, False,  True,  True,  True,  True, False, False,  True,
        True, False,  True,  True,  True,  True, False,  True, False,
        True, False, False,  True, False,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True])

sherpa In [9]: (dy1 == dy2).all()
Out[9]: True
```

Reply:
Note that I'm not sure this example is actually telling me what I think it's telling me...